### PR TITLE
error 404 resolved with a toast, topbar color made darker

### PR DESCRIPTION
**Pull request by derins**
This PR address issue #18

# Current issues
none

# Code changes
1. color for topbar made darker
2. new div added as toast below refresh button in html
3. toast is triggered in main.js

# Documentation changes
none

# Validation suggestions
1. run website and type a faulty input into any input box

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -54,6 +54,14 @@
                         <div class="form-group">
                             <button id='refresh-data' class="btn btn-primary" type="button">Refresh</button>
                         </div>
+                        <div class="toast" style="display:none">
```
Comment (jongoncalves) [inline src/index.html]: styles in the css, please don't put them inline

File: src/js/main.js [hunk 2]
```diff
@@ -127,7 +127,8 @@ var main = main || {};
             // attach the generated options to the dom
             ns.selectOptionEl.html(selectOptions);
         }).fail(function(err) {
-            // TODO: create error handler
+            $('.toast').attr('style','');
```
Comment (jongoncalves) [inline src/js/main.js:130]: Is this necessary?

Comment (derins) [reply src/js/main.js:130]: That is to get rid of the style element "display:none" when toast is triggered. The reason I put "display:none" is that there was some empty space where the div is located.

File: src/index.html (hunk 1 above)
Comment (derins) [reply src/index.html]: Fixed

**Review by jongoncalves — APPROVED** [review]

**Pull request merged**